CBLClient/Apps/CBLTestServer-Dotnet/TestServer.NetCore/TestServer.NetCore.csproj

```diff
@@ -15,7 +15,6 @@
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Scripting" Version="2.6.1" />
     <PackageReference Include="System.ServiceProcess.ServiceController" Version="4.5.0-preview1-25914-04" />
     <PackageReference Include="Couchbase.Lite.Enterprise.Support.NetDesktop" Version="2.1.0-b0232" />
-    <PackageReference Include="Couchbase.Lite.Enterprise" Version="2.1.0-b0232" />
```

Comment:
no need to check in this file? Can you remove this checkin?

Reply:
It's required because jenkins job uses this file to build the app properly and because the appropriate packages were not added or extra packages were added, the jobs fails to build app.  This modification is required.